tests/e2e/features/3DViewer.feature

```diff
@@ -0,0 +1,43 @@
+Feature: preview 3D model
+  As a user
+  I want to preview a 3D model
+  So that I can make sure this is the correct file (e.g. before sharing it with others)
+
+  Background:
+    Given the user has logged in with username "admin" and password "admin"
+    And a 3D model (.glb file) has been uploaded
+    #should the filename be specified here?
+
+  Scenario: preview 3D model
```

Comment:
Since this is a very small/minimal feature, I would prefer to do the journey-style scenario (user performs all the actions in a single scenario)

Reply:
sounds great, can you explain me more about this type of scenario? shall we chat about that in person?